src/main/resources/modals/config.jelly

```diff
@@ -20,7 +20,7 @@
 
                     <h4 class="modal-title" id="configModalLabel">
 
-                        <img src="https://www.jenkins.io/images/logos/JCasC/JCasC.png" width="15%" height="15%"/>
+                        <img src="${resURL}/plugin/pull-request-monitoring/icons/JCasC.png" width="15%" height="15%"/>
```

Comment:
Wouldn't it make more sense to use the SVG (https://www.jenkins.io/artwork/)?

Reply:
Yes thats a good point. I will replace this image and also the plugin icons with svgs.